### PR TITLE
tasks sort by due date newest first

### PR DESCRIPTION
**Pull request by naienko**

### CODE REVIEW

File: src/components/ApplicationViews.js [hunk 1]
```diff
@@ -150,15 +150,12 @@ class ApplicationViews extends Component {
...
-
-
 			.then(() => this.setState(newState))
 	}
 
 	render() {
 
-		console.log(this.props.activeUser)
+		console.log(this.state)
 		return <React.Fragment>
```
Comment (katfree) [inline src/components/ApplicationViews.js:159]: believe quite a bit of routes are missing

Comment (katfree) [reply src/components/ApplicationViews.js:159]: jk....just testing you

**Review by katfree — APPROVED** [review]

**Comment by RDilly** [comment]
:shipit:

**Review by BrianBNeal — APPROVED** [review]
k

**Comment by RDilly** [comment]
I approve

**Pull request merged**